ark/utils/metacluster_remap_gui/colormap_helper.py

```diff
@@ -0,0 +1,29 @@
+import colorsys
+import itertools
+
+import matplotlib
+
+
+def distinct_cmap(n=33):
+    rgbs = distinct_rgbs(n)
+    return matplotlib.colors.ListedColormap(rgbs)
+
+
+def distinct_rgbs(n=33):
```

Comment:
The other helper functions don't need a docstring since they're so short, but I think it would be useful to have one for `distinct_rgbs`

Reply:
Agreed  :+1: